skopt/optimizer/base.py

```diff
@@ -263,20 +263,32 @@ def base_minimize(func, dimensions, base_estimator,
 
         models.append(gp)
 
-        if acq_optimizer == "sampling":
+        if acq_optimizer == "sampling" or acq_optimizer == "spearmint":
             X = space.transform(space.rvs(n_samples=n_points,
                                           random_state=rng))
             values = _gaussian_acquisition(
                 X=X, model=gp,  y_opt=np.min(yi), acq_func=acq_func,
                 xi=xi, kappa=kappa)
             next_x = X[np.argmin(values)]
 
-        elif acq_optimizer == "lbfgs":
+        if acq_optimizer == "lbfgs" or acq_optimizer == "spearmint":
             best = np.inf
 
-            for j in range(n_restarts_optimizer):
-                x0 = space.transform(space.rvs(n_samples=1,
-                                               random_state=rng))[0]
+            if acq_optimizer == "lbfgs":
+                x0_all = space.transform(space.rvs(n_samples=n_restarts_optimizer,
+                                         random_state=rng))
+            else:
+                std_ = 1e-3
+                best_X = Xi[np.argmin(yi)]
+
+                X_spray = best_X + rng.randn(n_spray_points, space.transformed_n_dims)
```

Comment:
We should scale the std used by `randn` to match the bounds of each dimension or `space.transform` the samples before adding them to `best_X`